v1.0/v1.0/bwa-mem-tool.cwl

```diff
@@ -5,8 +5,6 @@ cwlVersion: v1.0
 class: CommandLineTool
 
 hints:
-  - class: ResourceRequirement
-    coresMin: 4
```

Comment:
@mr-c I agree, using data from resource requirement is part of the test, but it should be lower, because it prevents running conformance tests on 2-core amazon instances, which is silly.